### DISCUSSION TITLE
Multilanguage template support

### DISCUSSION THREAD

**Discussion by francescbassas** [voidlabs/mosaico#635] [Q&A]
We recently start using Mosaico integration with CiviCRM (see https://github.com/veda-consulting/uk.co.vedaconsulting.mosaico). It's amazing! We customized an organization template based on versafix-1.

A small "inconvenience" that we are finding is that to define the same template in different languages (we work with (catalan, spanish and english) we have to replicate it for each language and change language related texts, links and images. This complicates the development of the template, but also the preparation of mailings because we have to start from scratch every time we make the same mailing from different languages.

It would be great that Mosaico would allow the language of the template to be defined as a variable and depending on it, it could modify texts, images and links, for example.

**Comment by mrecamales** [comment]
+1

**Comment by bago** [comment]
Have a look at this: https://github.com/voidlabs/mosaico/issues/308

**Comment by mrecamales** [comment]
Hi, 
We have developed a small application to deal with the different languages in templates amb make it easy to customize it. Have a look at: https://github.com/babu-cat/mosaico-templating-tool
